docs/dev/pass_infra.rst

```diff
@@ -389,6 +396,103 @@ To allow other C++ modules to apply this pass, we declare a free function in
 
     TVM_DLL Pass FoldConstant();
 
+.. _pass_instrument_section_tag:
+
+Pass Instrument
+~~~~~~~~~~~~~~~
+
+``PassInstrument`` provides callbacks run when entering/exiting ``PassContext`` and before/after executing passes.
```

Comment:
Since PassInstrument is an interface, it might be more helpful to speak about the interface as a whole rather than mention callbacks, which are just one way of implementing the interface. e.g.

The PassInstrument interface allows you to run code when entering/exiting PassContext and before ...

Reply:
Done.